crates/rpc/src/v06/method/get_transaction_receipt.rs

```diff
@@ -0,0 +1,61 @@
+use crate::context::RpcContext;
+use crate::v05::method::get_transaction_receipt as v05;
+
+pub async fn get_transaction_receipt(
+    context: RpcContext,
+    input: v05::GetTransactionReceiptInput,
+) -> Result<v05::types::MaybePendingTransactionReceipt, v05::GetTransactionReceiptError> {
+    // v0.5 has a different fee structure, but that gets handled in the v0.5 method. We can
+    // safely use the impl as is.
```

Comment:
If `get_transaction_receipt_impl()` follows v06 semantics shouldn't we move the actual implementation here?

Reply:
Sure will do; at the time I wanted to minimize the diff.